### PR TITLE
Prices change Indicator

### LINKED ISSUE #1840: Daily price change on the TokenList screen
[Experiment proposal
](https://www.notion.so/valora-inc/Daily-changes-in-token-prices-on-the-Token-List-screen-6e228ab5728f4f43a34d4bdd9b88f171)
[Design](https://www.figma.com/file/8USELhisKuP1YlCt95WrD2/Support-for-Multiple-Tokens?node-id=1713%3A60226)

AC:
- [ ] The daily price changes are displayed on TokenList screen;

- [ ] Experiment is setup in firebase

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

This PR add a price change indicator in the TokenBalanceScreen.
This new feature is behind the ff `showPriceChangeIndicatorInBalances`

### Tested

Locally in an iOS emulator

### How others should test

Override `showPriceChangeIndicatorInBalances` ff to turn on the feature, then go to TokenBalances Screen, you should be able to see the price change indicators.

### Related issues

- Fixes #1840

### Screenshots
Simulator Screen Shot - iPhone 12 - 2022-03-08 at 12 30 21

### CODE REVIEW

File: packages/mobile/src/components/PercentageIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  testID?: string
+  comparedValue: BigNumber.Value
+  currentValue: BigNumber.Value
+}
+
+function renderPercentage(percentage: BigNumber, testID?: string) {
+  const comparison = percentage.comparedTo(0)
+  const percentageString = percentage.abs().toFixed(2)
+  let style
+  let indicator: any
```
Comment (gnardini) [inline packages/mobile/src/components/PercentageIndicator.tsx:19]: nit: `let style: TextStyle` and `let indicator: React.ReactElement`

File: packages/mobile/src/components/PercentageIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    </View>
+  )
+}
+
+function PercentageIndicator({ comparedValue, currentValue, testID }: Props) {
+  const comparedValueBN = new BigNumber(comparedValue)
+  const currentValueBN = new BigNumber(currentValue)
+  const percentage = currentValueBN.dividedBy(comparedValueBN).multipliedBy(100).minus(100)
+
+  return renderPercentage(percentage, testID)
```
Comment (gnardini) [inline packages/mobile/src/components/PercentageIndicator.tsx:48]: I guess this is subjective, but I like to draw the main UI in the main screen function and only move logic to other 

So what I'd do in this case is something like:

```
function PercentageIndicator({ comparedValue, currentValue, testID }: Props) {
  const { indicator, percentageString } = usePercentageInfo(comparedValue, currentValue)

  return (
    <View style={{ flexDirection: 'row' }} testID={testID}>
      <View style={{ padding: 6 }}>{indicator}</View>
      <Text style={style}>{percentageString}%</Text>
    </View>
  )
}
```

and the hook would be very similar to the current `renderPercentage` (I just moved the `BigNumber` wrapping to it).

Changing this is optional of course, I was just sharing another possible approach :)

**Review by gnardini — APPROVED** [review]
Amazing job! 🔥

**Comment by ValoraQA** [comment]
Hey @MuckT please help us how to test this PR as it would require to enable a feature from backend 
Thanks !

**Pull request merged**